tests/end_to_end/test_validation_end_to_end.py

```diff
@@ -185,29 +186,26 @@ def call_validate_all(pr=None):
     featurename = 'Z_1'
     submit_feature(repo, contrib_dir, username, featurename, new_feature_str)
 
-    # if we expect this feature to succeed -- with NoOpAcceptanceEvaluator
-    call_validate_all(pr=2)
+    # if we expect this feature to fail
+    with pytest.raises(CalledProcessError):
+        call_validate_all(pr=2)
 
     # if we expect this feature to fail -- with a more reasonable evaluator
     # with pytest.raises(CalledProcessError):
     #     logger.info('Validating pull request 2, User Charlie, Feature Z_1')
     #     call_validate_all(pr=2)
 
-    # write another new feature
+    # write another new feature - redudancy
     repo.git.checkout('master')
     switch_to_new_branch(repo, 'pull/3')
-    new_feature_str = make_feature_str('B')
+    new_feature_str = make_feature_str('A')
     username = 'charlie'
-    featurename = 'B'
+    featurename = 'A'
     submit_feature(repo, contrib_dir, username, featurename, new_feature_str)
-    call_validate_all(pr=3)
+    with pytest.raises(CalledProcessError):
+        call_validate_all(pr=3)
 
-    # merge PR with master
```

Comment:
why remove this part of the test?

Reply:
I changed this from an accepted feature to a redundant feature that should not be accepted. I can add back the original feature B as well.

Reply:
ah, no i get it